scoped/post/create-post.md

```diff
@@ -0,0 +1,46 @@
+**Name**
+Create post
+
+----
+
+**Actor** 
+Signed-in user
+
+----
+
+**Subject area**
+Any
+
+----
+
+**Preconditions** 
+User is signed in.
+
+----
+
+**Termination outcome**
+User is on question/post page for the now-posted item.
+
+----
+
+**Basic flow**
+
+- User clicks on "ask question" / "post" / etc button (name can vary by category).
+
+- New page has:
+  - for first-time poster, intro message w/link to help (just-in-time help; can be simple or omitted for MVP)
+  - "New post in (category name)" - header to reinforce where you are
```

Comment:
How does the user assign the post to a category?

Reply:
The "post" button on any category page is to post in that category.  Later we should talk about whether that's selectable from the "post" page, but we'll also need to (potentially) let the person choose *type* of post (e.g. meta question vs discussion), so we should consider those cases together, after MVP.